x-pack/plugins/observability/public/pages/rule_details/index.tsx

```diff
@@ -323,29 +323,37 @@ export function RuleDetailsPage() {
                     defaultMessage: 'Alerts',
                   })}
                 </ItemTitleRuleSummary>
-
-                <ItemValueRuleSummary
-                  extraSpace={false}
-                  itemValue={`
+                {isLoadingLast24hAlerts ? (
```

Comment:
@fkanout I suggest you move this into a `useEffect` otherwise the whole page could break, similar to how it broke here https://github.com/elastic/kibana/pull/132491

Reply:
@mgiota, this is a different use case. The page will not break even if the `Last24hAlerts` hook throws an error. The page will keep working as usual and load everything EXCEPT the last-24h-alerts will fail silently. As we agreed, we will discuss handling the error in our TT.

Reply:
Since nothing break with the testing you did, I can approve to not block you with this PR. If needed, we can create a follow up PR.